spyder/plugins/editor/widgets/completion.py

```diff
@@ -253,14 +257,42 @@ def update_list(self, filter_text):
         else:
             self.hide()
 
+    def set_item_text(self, item_widget, item_info, height, width):
+        """Set item text using the info available."""
+        item_provider = item_info['provider']
+        item_type = self.ICONS_MAP.get(item_info['kind'], 'no_match')
+        item_label = item_info['label']
+        icon_provider = ("iVBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0l"
+                         "EQVR42mNkYAAAAAYAAjCB0C8AAAAASUVORK5CYII=")
+        img_height = height - 2
+        img_width = img_height * 0.8
+
+        if item_provider == KITE_COMPLETION:
+            icon_provider = ima.get_kite_icon()
```

Comment:
The icon appears to be blurry due to the resizing being done by the GUI.

![image](https://user-images.githubusercontent.com/3752225/65712605-41590480-e04c-11e9-882c-264719b5892d.png)

You can fix this by resizing the image here:

```python
kite_height = img_height
kite_width = (416.14/526.8) * kite_height
icon_provider = ima.get_kite_icon()
icon_provider = ima.base64_from_icon_obj(
    icon_provider, kite_width, kite_height)
```

This is what it looks like when manually resized:

![image](https://user-images.githubusercontent.com/3752225/65712556-29818080-e04c-11e9-8f91-4431e849c7a8.png)